### PR TITLE
Writing a New SQLFlow Interpreter

### LINKED ISSUE #16: Write a new interpreter
After understanding the architecture of the zeppelin-sqlflow plug-in and important code files, how to contributea new Interpreter to Zeppelin releases? You can refer to `Writing a New SQLFlow Interpreter.md`. Through a simple case, to get familiar with the creation process of an interpreter.

### PR DESCRIPTION
**Pull request by Klay1124**
fix #16

Through a simple case, to get familiar with the creation process of an interpreter.

### CODE REVIEW

File: Writing a New SQLFlow Interpreter.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+﻿## 1. What is Apache Zeppelin Interprer
+**Apache Zeppelin Interpreter** is a language backend. For example to use sqlflow code in Zeppelin, you need a **sqlflow** interpreter. We integrate the open source machine learning tool SQLFlow of Ant Group into the web-based notebook tool Apache Zeppelin, so as to expand the interpreter language supported by Zeppelin.
```
Comment (weiguoz) [inline Writing a New SQLFlow Interpreter.md:5]: Add a blank line between topic and text.

**Review by weiguoz — CHANGES_REQUESTED** [review]

File: Writing a New SQLFlow Interpreter.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+## 1. What is Apache Zeppelin Interprer
```
Comment (weiguoz) [inline Writing a New SQLFlow Interpreter.md:3]: Need a `topic`.

File: Writing a New SQLFlow Interpreter.md (hunk 1 above)
Comment (Klay1124) [reply Writing a New SQLFlow Interpreter.md:5]: OK.

File: Writing a New SQLFlow Interpreter.md (hunk 2 above)
Comment (Klay1124) [reply Writing a New SQLFlow Interpreter.md:3]: The topic has been added.

**Review by weiguoz — APPROVED** [review]
LGTM

**Pull request merged**